config/application.rb

```diff
@@ -51,6 +52,7 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :sidekiq
 
+    config.middleware.insert_after ActionDispatch::RequestId, RedirectToServiceGovUkMiddleware
```

Comment:
Is this the earliest it could possibly go?

Reply:
> Looks good — keen to see this on QA.
> 
> Could we perhaps only insert the middleware on qa and staging as

Deployed it to QA last night for testing, can do a deploy again if you'd like to take a look.

Reply:
@duncanjbrown deployed it to QA https://qa.apply-for-teacher-training.education.gov.uk/ (should take you to service.gov.uk)
have also inserted the middleware after `ActionDispatch::HostAuthorization` will try inserting before next

Reply:
I think its better to leave it after https://edgeapi.rubyonrails.org/classes/ActionDispatch/HostAuthorization.html ?

Reply:
configured it after `ActionDispatch::HostAuthorization` as it protects against unauthorised hosts.